lib/rules/block-indentation.js

```diff
@@ -552,7 +555,7 @@ export default class BlockIndentation extends Rule {
         const elseLine = node.program.loc.end.line;
         const fixedNode = this.fixLine(
           lines,
-          elseLine - 1,
+          elseLine - (path ? 1 : node.loc.start.line),
```

Comment:
Took me some time to find, but this is basically similar to for example line 498 above. At first I didn't have the `path ? 1` part in there, but I added an example test that failed without this :+1: